aten/src/ATen/native/RNN.cpp

```diff
@@ -1496,6 +1512,10 @@ std::tuple<Tensor, Tensor> lstm_cell(
     const Tensor& input, TensorList hx,
     const Tensor& w_ih, const Tensor& w_hh, const Tensor& b_ih, const Tensor& b_hh) {
   TORCH_CHECK(hx.size() == 2, "lstm_cell expects two hidden states");
+  check_rnn_cell_forward_input(input, w_ih.size(1));
+  auto hidden_size = w_hh.size(1);
+  check_rnn_cell_forward_hidden(input, hx[0], hidden_size, "[0]");
```

Comment:
avoid string args, they aren't needed unless error message has to be constructed (unlikely), this argument can be `int`